### PR TITLE
dpg.danawa.com

### PR DESCRIPTION
**Pull request by green1052**
https://dpg.danawa.com/news/view?listSeq=5076046&boardSeq=61

### CODE REVIEW

File: filters-share/specific_ELEMHIDE.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ newsen.com,toomics.com,m.ytn.co.kr,djtimes.co.kr,mdjournal.kr,zum.com,windowsfor
 ! ##div[class^="ad"]
 ppomppu.co.kr,m.newsen.com,peterpanz.com,jwmx.tistory.com,thespike.co.kr,hobbyen.co.kr,socialvalue.kr,entertainpost.com,m.moneys.mt.co.kr,basketkorea.com,m.ohmynews.com,zdnet.co.kr,biz.heraldcorp.com,slrclub.com,sports.chosun.com##div[class^="ad"]
 ! ##div[class^="banner_"]
-auth.danawa.com,socialvalue.kr,news.jtbc.joins.com,etoland.co.kr,ajunews.com,mk.co.kr##div[class^="banner_"]
+prod.danawa.com,auth.danawa.com,socialvalue.kr,news.jtbc.joins.com,etoland.co.kr,ajunews.com,mk.co.kr##div[class^="banner_"]
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt:16]: What is this?

**Comment by green1052** [comment]
image

how about script + div[style]

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_ELEMHIDE.txt (hunk 1 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt:16]: leftover

Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt:16]: image

like

Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt:16]: check here

http://prod.danawa.com/info/?pcode=16101083

**Comment by piquark6046** [comment]
https://github.com/List-KR/List-KR/pull/427#issuecomment-1166299096

I disagree because AdGuard installable program filters the HTML element.

<details>
<summary>Screenshot</summary>

Screenshot from 2022-06-25 14-33-35


</details>

**Comment by green1052** [comment]
> Referen

no script tag, see under 

**div[style="min-..."**

its letfover

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**